applications/EVMNS.md

```diff
@@ -0,0 +1,207 @@
+| 0a.  | License  | MIT License  |
+| 0b.  | Documentation  | Provide and update documentation for this milestone in the form of instruction files and examples in the repository, provide guidance in the Readme file, and more in-depth guidance in our articles. |
+| 0c.  | Testing Guide  | The functions of this milestone can be tested by running the unit tests we provide, and guidance on how to run these tests is provided in the Readme file.  |
+| 0d.  | Docker  | Use Docker to start a container that runs a local nodevm instance where specific users and contracts are deployed for developer testing. This ensures that developers have the same environment when using EVMNS.  |
+| 1  | In-depth test  | Automated unit tests with 100% coverage and multiple rounds of functional testing are completed internally to ensure functionality and robustness.  |
+| 2  | dApp Development Documentation  | Establish a perfect dApp development documentation system to meet and support the quick access of dApp developers.  |<br/><br/>
+### Milestone 5 - Deployment Test Network
+- **Estimated duration:** 1 weeks
+- **FTE:** 2
+- **Costs:** 10,000 USD
+
+| ID  | Deliverable  |Specification|
+| ------------ | ------------ | ------------ |
+| 0a.  | License  | MIT License  |
+| 0b.  | Documentation  | Provide and update documentation for this milestone in the form of instruction files and examples in the repository, provide guidance in the Readme file, and more in-depth guidance in our articles. |
+| 0c.  | Testing Guide  | The functions of this milestone can be tested by running the unit tests we provide, and guidance on how to run these tests is provided in the Readme file.  |
+| 0d.  | Docker  | Use Docker to start a container that runs a local nodevm instance where specific users and contracts are deployed for developer testing. This ensures that developers have the same environment when using EVMNS.  |
+| 0e. | Article| We will publish an English article about the project on medium.com, explaining how smart contracts work and how the project will be used. Also, we will illustrate the goals and outcomes set through the grant.|
+| 1  | New version of the official website  | Including project introduction, domain registration and management, instruction documents, etc.  |
+| 2  | Deployment of test networks  | Deploy the test environment and conduct public testing to further ensure the smooth launch and security of the system  |<br/><br/>
+
+#### Multi-Chain Ecology
+Multi-chain ecology is the focus of the second phase of development, and we will prioritize the completion of the first phase of development (the above five milestones) to maximize the assurance that EVMNS can be launched and run stably at the first time after the official launch of EVM.<br/><br/>
+Therefore, the development content, duration and cost of the multi-chain ecology will be separately defined before the launch of the second phase.<br/><br/>
+# Future Plans
```

Comment:
I see that milestones stop at testnet, and there are no future plans for mainnet deployment. 
Can you elaborate on what the plans are for mainnet please?

Reply:
Hi Nathan,

Thanks for your reply.

The future plan for mainnet deployment has been added, the details are as following：

Miltiple rounds of in-depth testing will be carried out first after the testnet is deployed. When all the problems found in the test are fixed, we will start the security audit of the smart contract to further ensure security. On the premise that the security audit is completely passed, it will finally be deployed to the mainnet and the project will be officially launched.

It is estimated that it will take about 1 month to complete the above items.

Reply:
Thanks!